hadoop-hdfs-project/hadoop-hdfs-rbf/src/test/java/org/apache/hadoop/hdfs/server/federation/fairness/TestRouterHandlersFairness.java

```diff
@@ -208,4 +212,15 @@ private void invokeConcurrent(ClientProtocol routerProto, String clientName)
     routerProto.renewLease(clientName);
   }
 
+  private int getTotalRejectedPermits(RouterContext routerContext) {
+    int totalRejectedPermits = 0;
+    for (String ns : cluster.getNameservices()) {
+      totalRejectedPermits += routerContext.getRouter().getRpcServer()
```

Comment:
We may want to extract:
routerContext.getRouter().getRpcServer().getRPCClient()

Reply:
Updated.